vendor.conf

```diff
@@ -22,9 +22,9 @@ github.com/docker/go-units v0.3.1
 github.com/docker/spdystream 449fdfce4d962303d702fec724ef0ad181c92528
 github.com/emicklei/go-restful v2.2.1
 github.com/godbus/dbus v3
-github.com/gogo/googleapis 08a7655d27152912db7aaf4f983275eaf8d128ef
-github.com/gogo/protobuf v1.0.0
-github.com/golang/protobuf v1.1.0
+github.com/gogo/googleapis v1.2.0
```

Comment:
These just match cd?

Reply:
Yes, I updated a few vendorings but it was always to match containerd

Reply:
Deal thanks